docs/migrate-to-jina3/migrate.md

```diff
@@ -0,0 +1,72 @@
+# Migration to Jina 3
```

Comment:
Are we going with `3` or `3.0`?

Reply:
I think 3 is better, assuming that we don't have more breaking changes any time too soon